balloc/balloc.c

```diff
@@ -640,9 +652,13 @@ balloc_group_write_credit(struct m0_balloc               *bal,
                           struct balloc_groups_write_cfg *bgs,
                           struct m0_be_tx_credit         *credit)
 {
-	m0_be_btree_insert_credit(&bal->cb_db_group_extents, 2,
-		M0_MEMBER_SIZE(struct m0_ext, e_start),
-		M0_MEMBER_SIZE(struct m0_ext, e_end), credit);
+	int i;
+
+	for (i = 0; i < bal->cb_sb.bsb_indexcount; i++) {
```

Comment:
I have a doubt about this credit calculation.
I am not sure if all the btree (cb_db_group_extents[]) are needed.
Please review the code.

Reply:
Please correct me if i am wrong here, Following function balloc_group_write_credit() will be called during mkfs, this is onetime job, where we need to create multiple btrees[No of indexes]. We need  indexcount * No of btree credits here. where btree db formatting will happen.